LayoutTests/http/wpt/html/dom/scroll-to-text-fragment/resources/iframe-scroll-to-text-fragment.html

```diff
@@ -0,0 +1,5 @@
+<html>

```

Comment:
You don't need any of these tags!

Reply:
Tests under http/wpt run with the web-platform-tests infrastructure (use wpt http server and testharness.js) but are not upstreamed and can be browser-specific if necessary.

That said, it is always a good idea to have tests that work across browsers whenever possible.

Reply:
Good to know!